datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -134,12 +130,12 @@ func (f *ChainDataFetcher) Stop() error {
 	return nil
 }
 
-func (f *ChainDataFetcher) makeRequests(start, end uint64, reqType requestType, stopCh chan struct{}) {
+func (f *ChainDataFetcher) makeRequests(start, end uint64, reqType requestType, updateCheckpoint bool, stopCh chan struct{}) {
```

Comment:
Using both `newRequest` and `makeRequests` confuses me. 
How about changing `makeRequests` to `sendRequests`?
```suggestion
func (f *ChainDataFetcher) sendRequests(start, end uint64, reqType requestType, updateCheckpoint bool, stopCh chan struct{}) {
```